src/main/java/ui/components/KeyboardShortcuts.java

```diff
@@ -130,10 +131,17 @@
     public static final KeyCodeCombination CLOSE_PANEL =
             new KeyCodeCombination(KeyCode.W, KeyCombination.CONTROL_DOWN);
     
-    private static Set<KeyCodeCombination> populateJumpToFirstIssueSet() {
-        Set <KeyCodeCombination> result = new HashSet<KeyCodeCombination>();
-        result.add(new KeyCodeCombination(KeyCode.DIGIT1));
-        result.add(new KeyCodeCombination(KeyCode.DOWN, KeyCombination.CONTROL_DOWN));
+    private static Map<Integer, KeyCodeCombination> populateJumpToNthIssueMap() {
+        Map<Integer, KeyCodeCombination> result = Collections.unmodifiableMap(new HashMap<>());
```

Comment:
This won't work -- the point of an unmodifiable map is that it's immutable. You have to make the map unmodifiable after putting the key combinations in, before returning.